solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpJdkSolrClient.java

```diff
@@ -553,12 +555,6 @@ public Builder(String baseSolrUrl) {
 
     @Override
     public HttpJdkSolrClient build() {
-      if (idleTimeoutMillis == null || idleTimeoutMillis <= 0) {
-        idleTimeoutMillis = (long) HttpClientUtil.DEFAULT_SO_TIMEOUT;
-      }
-      if (connectionTimeoutMillis == null) {
-        connectionTimeoutMillis = (long) HttpClientUtil.DEFAULT_CONNECT_TIMEOUT;
-      }
```

Comment:
same comment as for Http2SolrClient: these settings were redundant/duplicative and thus misleading.  Only need to be inside the underlying HttpClient.